app/config/env.php

```diff
@@ -82,3 +82,20 @@
     $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/cache_pool'));
     $loader->load($pool . '.yml');
 }
+
+// Yoochoose settings
+if ($value = getenv('YOOCHOOSE_CUSTOMER_ID')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.customer_id', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_LICENSE_KEY')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.license_key', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_SERVER_URI')) {
+    $container->setParameter('ez_recommendation.default.server_uri', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_INCLUDED_CONTENT_TYPES')) {
```

Comment:
Looks ok, but I'd prefer if we skip YOOCHOOSE_INCLUDED_CONTENT_TYPES, and instead you set a value that makes sense for studio demo in default_parameters.yml. As I said it's a setting more tied to what kind og demo and demo content you install, not to your subscription.